### ISSUE TITLE
column 'recording_date' is incorrect

### ISSUE THREAD

**Issue by jakewilson** [bb4242/dharmaseed-android#30]
The `recording_date` column in the table `talks` is incorrect.
We insert values into `talks` based on the info we get from the api. So if there is a `recording_date` in the talk json from the api, we insert that into the db. But in the api, the field is `rec_date`, so `recording_date` doesn't get inserted. Since we don't have a `recording_date`, we use the `update_date`, which causes the most recent talks in the NavigationActivity to often be from one teacher because that teacher got updated last of all. For example:
![brian](https://user-images.githubusercontent.com/3004336/50741703-c8203580-11ce-11e9-9c41-7c540bee1704.png)

Looking at the returned json for the talk "The Kalama Sutta":

{"edition": "2019-01-06 21:02:07", "items": {"54055": {"upload_date": "2019-01-04 00:00:00", "is_featured": false, "audio_url": "/teacher/484/talk/54055/20170724-Brian_Lesage-FIMC-the_kalama_sutta-54055.mp3", "original_audio_file_name": "uploads/7_24_17_THE_KALAMA_SUTTA___NAVIGATING_THE_PATH.MP3", "id": 54055, "description": "This talk offers reflections on the Kalama sutta that make this discourse relevant to our lives.", "update_date": "2019-01-04 22:44:25", "venue_id": 256, "title": "The Kalama Sutta", "is_not_sellable": false, "destination": "web", "language_id": 1, "recording_type": "Talk", "duration_in_minutes": 30.7713333333333, "**rec_date**": "2017-07-24 19:00:00", "is_album": false, "retreat_id": 4129, "tracks": [], "publishability": "general", "teacher_id": 484, "isNotSellable": false, "recording_date_in_seconds": 1500922800.0, "talk_code": "", "isAlbum": false}}}

We see that the key is `rec_date` and that the `rec_date` is "2017-07-24 19:00:00", but the `update_date` is "2019-01-04 22:44:25", so it appears near first in the list of talks.

Check out these two queries:
```
sqlite> select count(_id) from talks where recording_date IS NOT NULL;
0
```

```
sqlite> select count(_id) from talks where recording_date IS NULL;
32022
```

So a simple change in the DB from `recording_date` to `rec_date` should fix this.

**Comment by jakewilson** [comment]
Ah, I found the actual culprit. The `recording_date` column was incorrect, that's true, but we order the list view results here:
(https://github.com/bb4242/dharmaseed-android/blob/9fc41abc806867a45cee2c0c0258a6eb0a00a1bb/app/src/main/java/org/dharmaseed/androidapp/NavigationActivity.java#L740-L742)

@bb4242 is there a reason you made this `update_date` and not `rec_date`? I'd like to L742 change this to `DBManager.C.Talk.RECORDING_DATE`. It seems more natural, because teachers seem to do batch updates on a lot of their talks at once. Like right now I have about 20 or 30 talks all from Annie Nugent at the top of the list view, even though some of the talks are from 2014. It's just because all of her talks were updated today.
